### ISSUE TITLE
Incorrect vulnerability details

### ISSUE THREAD

**Issue by mNantern** [OSSIndex/vulns#57]
**Vulnerability URL**
Provide the URL to the OSS Index vulnerability. eg:
```
https://ossindex.sonatype.org/vuln/30aff9a6-cbf5-4fc7-9034-090e139fe1d8
```

**Description**
The current version of the package `perl` in Debian Stretch is `5.24.1-3+deb9u5` which is vulnerable to `CVE-2017-12814` [according to OSSIndex](https://ossindex.sonatype.org/component/pkg:deb/debian/perl@5.24.1%3Fdistro=stretch) altough it is not the case according to the Debian security tracker: https://security-tracker.debian.org/tracker/CVE-2017-12814

**Comment by ken-duck** [comment]
Sorry I have been so long in replying. We have been looking into the issue. This is rather a complicated situation. The official perl releases are vulnerable up to and including version 5.24.2

But Debian (and other Linux Distributions) will sometimes make patches to earlier versions, which is the case here.

I have been exploring some options. Neither is going to be super fast to implement. We are exploring the options and will hopefully be starting on a solution soon.

Thanks for bringing this to our attention.

**Comment by mNantern** [comment]
FYI Debian is publishing a json with all their CVE here: https://security-tracker.debian.org/tracker/data/json

It can be used to map a PURL to the CVE patched by Debian with small adjustments (the python deb is called `python` but it appears as `python2.7` in the json file).

**Comment by ken-duck** [comment]
Indeed. I have been looking at that, and it will prove most useful. Thanks! It will take some time before I can complete the ingest of the feed; but rest assured it is on the roadmap.
